packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -35,73 +39,125 @@ export const DBOT_ONBOARDING = [
     {
         target: 'body',
         content: (
-            <Step
-                label={localize('Start with a template')}
-                content={localize(
-                    'Load a template containing the Martingale, D’Alembert, or Oscar’s Grind strategy, and modify it as you wish.'
-                )}
+            <TourGuide
+                Label={localize('Get started on DBot')}
+                Content={localize('Hi [first name]! Hit Start for a quick tour to help you get started.')}
+                className={'dbot-onboarding__container'}
+                setJoyrideStepIndex={0}
             />
         ),
-        ...joyride_config,
+        ...joyride_props,
+    },
+    {
+        target: '#dc-tabs__content_group_tiles',
+        content: (
+            <TourGuide
+                Label={localize('Shortcuts')}
+                Content={localize('You can also use these shortcuts to import or build your bot.')}
+                className={'dbot-onboarding__container'}
+                setJoyrideStepIndex={0}
```

Comment:
index shouldn't be 1 ? the previous one is 0

Reply:
it should be dashboard tab index fixed ✅